pkg/kv/kvserver/kvflowcontrol/rac2/range_controller.go

```diff
@@ -1045,6 +1045,10 @@ func constructRaftEventForReplica(
 //
 // Requires replica.raftMu to be held.
 func (rc *rangeController) HandleRaftEventRaftMuLocked(ctx context.Context, e RaftEvent) error {
+	if (len(e.Entries) != 0 || e.Snap != nil) && e.Term != rc.term {
```

Comment:
Add a comment that `Term` can be empty when the event does not contain a storage append?

Reply:
Also, could why the term should match? The leader's log is consistent with itself. And the `rc.term` should have been updated up the stack already.

Reply:
Done
<!-- Reviewable comment -OC-e55Y7OergxVw7uSJ:bb6un1j -->
<!-- Sent from Reviewable.io -->